src/com/foamdev/demos/bootstrap/ControllerForms.js

```diff
@@ -0,0 +1,194 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'com.foamdev.demos.bootstrap',
+  name: 'ControllerForms',
+  extends: 'foam.u2.Element',
+
+  exports: [ 'as data' ],
+
+  imports: [ 'window' ], //To add bootstrap JavaScript
+
+  properties: [
+    {
+      class: 'String',
+      name: 'firstName',
+      view: {
+        class: 'foam.u2.TextField',
+        onKey: true
+      },
+      placeholder: 'First name',
+    //value:'Mark'
+    },
+    {
+      class: 'Boolean',
+      name: 'checkBoxTest',
+      value: true,
+      view: {
+        class: 'foam.u2.CheckBox',
+        onKey: true

```

Comment:
Is onKey used for CheckBox? If set it updates the model on every keystroke rather just when the user leaves the text field, but checkboxes don't have any incremental changes, the value just toggles between true and false when you click on it.